frontend/src/hooks/useStats.ts

```diff
@@ -0,0 +1,15 @@
+import { useState, useEffect, useContext } from 'react';
+import StatsContext from 'src/features/statistics/StatsContext';
+import { Statistics } from 'src/services/openapi';
+
+export const useStats = () => {
+  const [stats, setStats] = useState<Statistics>();
+
+  const { getStats } = useContext(StatsContext);
```

Comment:
It's possible to use `getStats()` to initialize the state, so that `stats` is never undefined:
```suggestion
  const { getStats } = useContext(StatsContext);
  const [stats, setStats] = useState<Statistics>(getStats());
```

Reply:
Should be fixed by https://github.com/tournesol-app/tournesol/pull/1542/commits/19e07570caca58f070b8e0eb4e39e716bf4627ed